e107_plugins/forum/forum_viewtopic.php

```diff
@@ -615,7 +616,7 @@ function forumbuttons($thread)
 
 if ($forum->prefs->get('enclose'))
 {
-	$forumTitle = e107::pref('forum','title', LAN_PLUGIN_FORUM_NAME);
+	$forumTitle = varset($FORUMCAPTION) ? $tp->parseTemplate($FORUMCAPTION, TRUE, $sc) : e107::pref('forum','title', LAN_PLUGIN_FORUM_NAME);
```

Comment:
Donne.
Thought that it was better to use the core varset function....
Is the varset core function to be deprecated?